CHANGELOG.md

```diff
@@ -5,10 +5,18 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## [master]
 ### Added
 - Client.withAuthAccess
+- The VacationResponse model. #45
+- Client.getVacationResponse
+- The SetResponse class
+- Client.setVacationResponse
 
 ### Fixed
 - New/changed JMAP endpoint properties in AuthAcces. #38
 
+### Removed
+- The MessagesSet class
```

Comment:
No they're all the same, it's a response to `setFoos` as per the spec. See http://jmap.io/spec.html#setfoos
It's true that some replies do not leverage all fields, bu they will just default and our code can remain coherent.